.github/workflows/scripts/prepare-new-issue.sh

```diff
@@ -45,14 +45,11 @@ for AREA in ${BODY_AREAS}; do
     continue
   fi
 
-  if [[ -n "${LABELS}" ]]; then
-      LABELS+=","
-  fi
-    LABELS+="${AREA}"
+  LABELS+=",${AREA}"
```

Comment:
oh, I didn't see this one! there's another `if [[ -n "${LABELS}" ]]` near the end of the file also

Reply:
good catch!